easybuild/framework/easyconfig/default.py

```diff
@@ -116,7 +116,7 @@
     'stop': [None, 'Keyword to halt the build process after a certain step.', BUILD],
     'tests': [[], ("List of test-scripts to run after install. A test script should return a "
                    "non-zero exit status to fail"), BUILD],
-    'unpack_options': [None, "Extra options for unpacking source", BUILD],
+    'unpack_options': ['', "Extra options for unpacking source", BUILD],
```

Comment:
a list would work too

`None` is not a list or a string, and so `self.cfg.update` doesn't work

We should also avoid using `[]` as a default value, since that's mutable... Those values should all be `None`, and then be later transformed into `[]`.